src/Messages.php

```diff
@@ -72,9 +72,16 @@ public function addMessage($key, $message)
         if (!isset($this->storage[$this->storageKey][$key])) {
             $this->storage[$this->storageKey][$key] = array();
         }
-        
+
         //Push onto the array
-        $this->storage[$this->storageKey][$key][] = (string)$message;
+        if (
+            is_array($message) === true
+            || is_object($message) === true
+        ) {
```

Comment:
@geggleto Sorry, I was meant to have added a comment about this with the PR.

This was a discussion point more than anything as my thoughts behind this was that pushing an array or object onto the `slimFlash` stack with the given key would result in the following:

```php
array(
    'slimFlash' => array(
        'key' => array(
            0 => array(
                'data' => array(
                    // ...
                )
            )
        )
    )
)
```

This could become annoying trying to access the `data` stack.

Am I over thinking the problem or is it just creating an issue/hindrance?